R/sample.R

```diff
@@ -209,11 +209,15 @@ st_sample_exact = function(x, size, ..., type) {
 	random_pt = st_sample(x = x, size = size, ..., type = type, exact = FALSE)
 	while (length(random_pt) < size) {
 		diff = size - length(random_pt)
-		random_pt_new = st_sample(x, size, ..., type, exact = FALSE)
+		random_pt_new = st_sample(x, size = diff, ..., type, exact = FALSE)
```

Comment:
I missed that somehow 1st time @edzer. Apologies. The result will be the same, just slightly slower I guess.